include/CLI/App.hpp

```diff
@@ -77,9 +77,15 @@ class App {
     /// If true, allow extra arguments in the ini file (ie, don't throw an error). INHERITABLE
     bool allow_config_extras_{false};
 
-    ///  If true, return immediately on an unrecognised option (implies allow_extras) INHERITABLE
+    ///  If true, return immediately on an unrecognized option (implies allow_extras) INHERITABLE
     bool prefix_command_{false};
 
+    /// set to true if an unnamed subcommand were added
+    bool has_unnamed_subcommands_{false};
```

Comment:
Can't this be queried? Not fond of adding too many values, and if we allow deleting/removing subcommands, this would be hard to track. Looping over a handful of subcommands is not too expensive.

Reply:
It is internal so wouldn't be queried, but if we did a couple more loops over the subcommands it could be removed